### ISSUE TITLE
Wrong metric --> Wrong accuracy

### ISSUE THREAD

**Issue by michael20at** [frlim/data2040_final#1]
Hi, I think your accuracy is wrong. If you use binary crossentropy, "acc" metric is only valid for binary classification problems. Cifar10 has 10 classes, you should use "categorical_accuracy" as metric, which is accurate, "acc" does not work correctly and reports wrong (too high) numbers in this case!

**Comment by michael20at** [comment]
Also, more of a question, why use binary cross entropy at all for cifar10, instead of categorical?

**Comment by AnanyaKumar** [comment]
Yup that's right, this should be fixed or added to the repository so people who find this repo on Google don't get the wrong numbers.
